recipe/meta.yaml

```diff
@@ -144,18 +144,17 @@ about:
     triangulations, in 2D, 3D, or more. It provides a substantial collection
     of generic, efficient and robust implementations of key algorithms in
     topological data analysis. It includes:
-    · For scalar data: critical points, integral lines, persistence diagrams,
-      persistence curves, merge trees, contour trees, Reeb graphs, Morse-Smale
-      complexes, topological simplification, topology-aware compression,
-      harmonic design;
-    · For bivariate scalar data: fibers, fiber surfaces, continuous
+    - For scalar data: critical points, integral lines, persistence diagrams,
+persistence curves, merge trees, contour trees, Reeb graphs, Morse-Smale complexes, topological simplification, topology-aware compression, harmonic design;
```

Comment:
All lines after `  description: |` need to be indented with the same number of spaces, so you probably need to add four spaces in front of this line.

Reply:
thanks a lot, I'm on it :)